lib/traits/HasChildViews.js

```diff
@@ -103,7 +103,7 @@ var HasChildViews = {
             return 0;
         }
 
-        return _.values(this.childViews).length;
+        return Object.keys(this.childViews).length;
```

Comment:
sure - algorithmically, if values is 2n (where n is the number of keys in the object), Object.keys is n because to get all the values for the object's keys, you need to find the keys first and then grab the value for each key.

For data, it's hard to do good benchmarks on my computer. i turned off as many background things as i could. here was the most recent runs of the current benchmark tests we have on my branch versus master respectively:

```benchmarks

Running "benchmark:all" (benchmark) task

Running suite Creating child views [benchmarks/creatingChildViews.js]...
>> creating child View with View constructor x 26,937 ops/sec ±3.13% (96 runs sampled)
>> creating child View with View constructor and options x 26,909 ops/sec ±2.61% (96 runs sampled)
>> creating child View with instance of a View x 27,218 ops/sec ±2.03% (98 runs sampled)
>> creating child View with an identifier x 27,101 ops/sec ±0.90% (98 runs sampled)
Fastest test is creating child View with an identifier at 1.01x faster than creating child View with View constructor and creating child View with View constructor and options and creating child View with instance of a View
```

vs

```
Running "benchmark:all" (benchmark) task

Running suite Creating child views [benchmarks/creatingChildViews.js]...
>> creating child View with View constructor x 24,678 ops/sec ±1.78% (97 runs sampled)
>> creating child View with View constructor and options x 24,202 ops/sec ±2.39% (95 runs sampled)
>> creating child View with instance of a View x 24,996 ops/sec ±0.92% (100 runs sampled)
>> creating child View with an identifier x 27,216 ops/sec ±2.28% (96 runs sampled)
Fastest test is creating child View with an identifier at 1.09x faster than creating child View with instance of a View
```

Creating child views calls childViewCount so these tests are testing it indirectly. I can make a benchmark for adding 100 child views to a view. I hypothesize that would make the difference clearer